src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -86,16 +94,41 @@ import {
   LevelItem,
   Stack,
   StackItem,
+  Switch,
   Text,
   TextContent,
   TextVariants,
+  Title,
   Toolbar,
   ToolbarContent,
   ToolbarGroup,
   ToolbarItem,
   Tooltip,
 } from '@patternfly/react-core';
-import { InfoCircleIcon, OutlinedQuestionCircleIcon, Spinner2Icon, TrashIcon } from '@patternfly/react-icons';
+import {
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  ExclamationTriangleIcon,
+  InfoCircleIcon,
+  OutlinedQuestionCircleIcon,
+  SearchIcon,
+  Spinner2Icon,
+  TrashIcon,
+} from '@patternfly/react-icons';
+import { css } from '@patternfly/react-styles';
+import {
+  InnerScrollContainer,
+  OuterScrollContainer,
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  ThProps,
+  Tr,
+} from '@patternfly/react-table';
+import { t } from 'i18next';
```

Comment:
Should import `t` from `useTranslation`?

Reply:
Ahh this is for the exported const. We probably should start exporting only translation keys and consumers somewhere can call `t`.